### DISCUSSION TITLE
FIPS and WPA3

### DISCUSSION THREAD

**Discussion by joelguittet** [openssl/openssl#24712] [Q&A] [answered]
Hello

I have read somewhere that WPA3 involve AES-SIV algorithms and because these algorithms are not FIPS validated, WPA3 indeed is NOT compatible with FIPS mode.
Is it correct ?

Thanks
Joel

**Comment by paulidale** [comment, marked as answer]
This is a question for your FIPS lab.

OpenSSL doesn't implement SIV mode in it's FIPS provider, so it cannot be compliant.

**Reply by joelguittet** [reply]
Thanks!
